assets/cubyz/biomes/cave/toadstool_big.zig.zon

```diff
@@ -0,0 +1,47 @@
+.{
+	.isCave = true,
+	.maxHeight = -512,
+	.minHeight = -48250,
+
+	.chance = 0.04,
+
+	.caves = -0.02,
+
+	.music = "cubyz:la_kubisa_caverna",
+	.stoneBlock = "cubyz:stone",
```

Comment:
this is now `cubyz:slate`.
Also as you long as you don't explicitly want to enforce a certain stone type, it's based to just leave it empty. This might allow us to overwrite the stone based on other factors in the future.